stdlib/3/configparser.pyi

```diff
@@ -19,7 +19,7 @@ _converters = Dict[str, _converter]
 _T = TypeVar('_T')
 
 if sys.version_info >= (3, 6):
-    _Path = Union[str, PathLike[str]]
+    _Path = Union[str, bytes, PathLike[str]]
 else:
     _Path = str
```

Comment:
Should this also be `Union[str, bytes]`?

Reply:
Actually, I think not. On the 3.6 branch of cpython, the line is
```python
if isinstance(filenames, (str, os.PathLike)):
```

If I'm reading [bpo-31307](https://bugs.python.org/issue31307) correctly, accepting `bytes` was added in 3.7.

I'll add a third branch of this conditional differentiating 3.7 and 3.6.